src/shims/unix/linux_like/eventfd.rs

```diff
@@ -217,7 +217,9 @@ fn eventfd_write<'tcx>(
 
             // The state changed; we check and update the status of all supported event
             // types for current file description.
-            ecx.epoll_send_fd_ready_events(eventfd)?;
+            // Linux seems to cause spurious wakeups here, and Tokio seems to rely on that
+            // (see <https://github.com/rust-lang/miri/issues/4374#issuecomment-3508873493>).
+            ecx.epoll_send_fd_ready_events(eventfd, /* force_edge */ true)?;
```

Comment:
@Darksonn this is the suspicious part... if I make this `false`, the [Tokio test](https://github.com/rust-lang/miri/issues/4374#issuecomment-3508873493) deadlocks....